src/Core/Constants.cs

```diff
@@ -5,6 +5,7 @@ public static class Constants
         public const int MaxAccounts = 5;
         public const string AndroidAppProtocol = "androidapp://";
         public const string iOSAppProtocol = "iosapp://";
+        public const string DefaultGenerated = "-";
```

Comment:
Rename to `DefaultUsernameGenerated`. We need to specify what is the thing that is default generated, it cannot be understood by context in this case.